app/src/main/java/com/dessalines/thumbkey/ui/components/keyboard/KeyboardKey.kt

```diff
@@ -367,96 +382,102 @@ fun KeyboardKey(
 
     // a 3x3 grid
     // Use box so they can overlap
+    // Some magic padding numbers so that large radii don'g obscure the ledends
+    val radiusPercent = keyRadius.toFloat() / 100.toFloat()
+    val yPadding = 0.dp + borderWidth.dp
+    val diagonalXPadding = lerp(legendPadding, 11.dp + borderWidth.dp, radiusPercent)
+    val diagonalYPadding = lerp(yPadding, 6.dp + borderWidth.dp, radiusPercent)
+
     Box(
         modifier = keyboardKeyModifier,
     ) {
         Box(
             contentAlignment = Alignment.TopStart,
             modifier = Modifier
                 .fillMaxSize()
-                .padding(horizontal = keyPadding),
+                .padding(horizontal = diagonalXPadding, vertical = diagonalYPadding),
```

Comment:
newlines here for vertical and horizontal imo. Even formatKotlin doesn't make things as pretty as they should.